dll/cpl/desk/lang/fr-FR.rc

```diff
@@ -7,26 +7,26 @@ CAPTION "Themes"
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "A theme is a background plus a set of sounds, icons, and other elements to help you personalize your computer with one click.", IDC_STATIC, 5, 5, 235, 30
-    LTEXT "Theme:", IDC_STATIC, 5, 42, 55, 10
+    LTEXT "Un thème est un fond d'écran et un ensemble de sons, icones et autres éléments vous permettant de personnaliser votre ordinateur en un clic.", IDC_STATIC, 5, 5, 235, 30
+    LTEXT "Theme :", IDC_STATIC, 5, 42, 55, 10
     COMBOBOX IDC_THEMES_COMBOBOX, 5, 52, 160, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON "Save &As...", IDC_THEMES_SAVE_AS, 170, 52, 70, 14
-    PUSHBUTTON "&Delete", IDC_THEMES_DELETE, 170, 70, 70, 14
+    PUSHBUTTON "Enregistrer sous...", IDC_THEMES_SAVE_AS, 170, 52, 70, 14
+    PUSHBUTTON "&Supprimer", IDC_THEMES_DELETE, 170, 70, 70, 14
     CONTROL "", IDC_THEMES_PREVIEW, "STATIC", SS_BITMAP, 5, 90, 235, 115, WS_EX_CLIENTEDGE
 END
 
 IDD_BACKGROUND DIALOGEX 0, 0, 246, 210
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Fond d'écran"
+CAPTION "Bureau"
 FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL "", IDC_BACKGROUND_PREVIEW, "Static", SS_OWNERDRAW, 70, 10, 105, 74, 0
     CONTROL "", IDC_BACKGROUND_LIST, "SysListView32", LVS_REPORT | LVS_SINGLESEL |
             LVS_NOCOLUMNHEADER | LVS_SHAREIMAGELISTS | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 8, 104, 174, 86
-    LTEXT "Sélectionnez une image à utiliser comme fond d'écran", IDC_STATIC, 8, 93, 230, 9
+    LTEXT "Arrière-plan :", IDC_STATIC, 8, 93, 230, 9
     PUSHBUTTON "&Parcourir...", IDC_BROWSE_BUTTON, 188, 104, 55, 15
     PUSHBUTTON "&Couleur...", IDC_COLOR_BUTTON, 188, 167, 55, 15
-    LTEXT "Position", IDC_STATIC, 188, 128, 55, 9
+    LTEXT "Position :", IDC_STATIC, 188, 128, 55, 9
```

Comment:
Spaces and semicolons (appears a lot of time, I won't make a comment each time it appears)

Reply:
In french, my understanding is that space is BEFORE colons. Am I wrong ? (let's discuss in Pm if required)